BASE/src/Microsoft.ApplicationInsights/Microsoft.ApplicationInsights.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <RootNamespace>Microsoft.ApplicationInsights</RootNamespace>
     <AssemblyName>Microsoft.ApplicationInsights</AssemblyName>
-    <TargetFrameworks>net452;net46;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net452;net46;net462;netstandard2.0</TargetFrameworks>
```

Comment:
We might want to add net462 to all dependent projects as well (like https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/WEB/Src/Web/Web/Web.csproj#L7)?

Otherwise, if userapp is net462 and refers microsoft.ai.web(which is only net452) will get microsoft.ai net452 only, and not net462.